content/en-us/luau/metatables.md

```diff
@@ -118,19 +118,19 @@ Here's the list of available metamethods:
 		  <td>__mode</td>
 		  <td>Used in weak tables, declaring whether the keys and/or values of a table are weak. **Note:** References to Roblox instances are never weak. Tables that hold such references will never be garbage collected.</td>
 		</tr>
-	  <tr>
-		  <td>__gc(table)</td>
-		  <td>Fired when the table is garbage-collected. **Note:** On Roblox, this metamethod is disabled.</td>
-		</tr>
```

Comment:
Hey @metatablecat, I think it's still worth leaving this in to give people some context in the (rare) case that they were expecting to see it. Maybe something like, `This metamethod is present in Lua, but removed in Roblox. Declaring it does nothing.` Thoughts?

Reply:
This seems like it could be an interesting choice, `__gc` is removed for security reasons so I feel like it would be better to lean it towards that as the reason